README.adoc

```diff
@@ -123,8 +123,12 @@ The following table lists all available configuration options.
 |JENKINSGRADLEPLUGIN_GRADLE_PLUGIN_REPOSITORY_URL|Gradle|The URL of the repository to use when resolving the Gradle Enterprise Gradle plugin and the Common Custom User Data Gradle plugin. Required if Jenkins agents are not able to access the Gradle Plugin Portal at https://plugins.gradle.org.
 |JENKINSGRADLEPLUGIN_GRADLE_ENTERPRISE_PLUGIN_VERSION|Gradle|Enables injection for Gradle builds and defines which version of the https://plugins.gradle.org/plugin/com.gradle.enterprise[Gradle Enterprise Gradle plugin] to use.
 |JENKINSGRADLEPLUGIN_CCUD_PLUGIN_VERSION|Gradle|Defines which version of the https://plugins.gradle.org/plugin/com.gradle.common-custom-user-data-gradle-plugin[Common Custom User Data Gradle Plugin] to use.
+|JENKINSGRADLEPLUGIN_GRADLE_INJECTION_ENABLED_NODES|Gradle|Comma separated list. Labels of nodes where Gradle injection should be enabled.
```

Comment:
```suggestion
|JENKINSGRADLEPLUGIN_GRADLE_INJECTION_ENABLED_NODES|Gradle|Comma separated list of node labels where the Gradle Enterprise Gradle plugin injection should be enabled.
```

Is there a default ? If so I'd add `By default all nodes are enabled` or similar. Please adapt all other parts as well in the readme also the Maven part.

Reply:
Done